src/components/form/form.js

```diff
@@ -109,7 +109,14 @@ class Form extends React.Component {
      * @default false
      */
     validateOnMount: PropTypes.bool,
-
+    /**
+     * If true, will disable the savebutton when clicked
+     *
+     * @property autoDisable
+     * @type {Boolean}
+     * @default false
+     */
+    autoDisable: PropTypes.bool,
```

Comment:
Would be tempted to make this true by default in the future although that would be a breaking change